lib/copy/__tests__/copy-case-insensitive-paths.test.js

```diff
@@ -3,7 +3,10 @@
 const assert = require('assert')
 const os = require('os')
 const path = require('path')
-const fs = require(process.cwd())
+const fs = require('../../')
+const platform = os.platform()
+
+console.log('platform: ' + platform)
```

Comment:
Is this kept intentional?

Reply:
Yes @JPeer264 I kept it for debugging purposes just in case!